src/mon/OSDMonitor.cc

```diff
@@ -1076,7 +1076,10 @@ void OSDMonitor::prime_pg_temp(
   const OSDMap& next,
   pg_t pgid)
 {
-  if (mon->monmap->get_required_features().contains_all(
+  if (mon->monmap->get_required_features().empty()) {
+    dout(10) << __func__ << ": quorum not formed" << dendl;
+    return;
+  } else if (mon->monmap->get_required_features().contains_all(
```

Comment:
@gregsfortytwo i ran into a crash in `OSDMonitor::get_trim_to()` while testing `TEST_mon_features()`, so i added all these guards.